demo/src/html/index.html.template

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="">
```

Comment:
Why not remove the attribute completely?

Reply:
Because an empty string means unknown language

Reply:
Actually, you are right, it should only be set to an empty string when you want to overrule a parent element to unknown.